src/snowflake/connector/aio/_network.py

```diff
@@ -827,6 +827,11 @@ async def _request_exec(
             RuntimeError,
             AttributeError,  # json decoding error
         ) as err:
+            if "Event loop is closed" in str(err):
+                logger.warning(
+                    f"Snowflake connection is never closed, which is causing {err}"
+                    "please use SnowflakeConnection.close() to close connection and save resources"
```

Comment:
I feel we can say something like "event loop is closed, the operation failed, if you see this message during when exiting program, it's likely that the connection doesn't get closed. Please close the connection when exiting.."  (use chatgpt to polish a little bit)